WalletWasabi.Fluent/ViewModels/NavBar/NavBarViewModel.cs

```diff
@@ -73,9 +73,8 @@ public void Activate()
 			})
 			.Subscribe();
 
-		this.WhenAnyValue(x => x.SelectedWalletModel)
-			.Select(model => Wallets.FirstOrDefault(w => w.WalletModel == model))
-			.BindTo(this, x => x.SelectedWallet);
+		this.WhenAnyValue(x => x.SelectedWallet!.WalletModel)
```

Comment:
SelectedWallet is nullable or not? If it is make a proper null check. If not then can we remove the nullable?